utils/coreutils/coreconsts.go

```diff
@@ -67,6 +67,8 @@ var (
 	BuildName   = "JFROG_CLI_BUILD_NAME"
 	BuildNumber = "JFROG_CLI_BUILD_NUMBER"
 	Project     = "JFROG_CLI_BUILD_PROJECT"
+	SigningKey  = "JFROG_CLI_SIGNING_KEY"
+	KeyAlias    = "JFROG_CLI_KEY_ALIAS"
```

Comment:
@oshratZairi,
Are the above 2 env vars used by the jfrog-cli-artifactory module?
If they are, then they should be defined there.